File: app/src/main/java/com/upclicks/ffcapp/base/BaseApp.java
package com.upclicks.ffcapp.base;

import android.app.Application;

import com.upclicks.ffcapp.session.SessionHelper;
import javax.inject.Inject;
import dagger.hilt.android.HiltAndroidApp;

@HiltAndroidApp
public class BaseApp extends Application {
    @Inject
    SessionHelper sessionHelper;
    @Override
    public void onCreate() {
        super.onCreate();
    }
}

File: app/src/main/java/com/upclicks/ffc/base/BaseApp.kt
package com.upclicks.ffc.base

import android.app.Application
import com.upclicks.ffc.commons.Keys
import com.upclicks.ffc.data.BaseURLConfigHelper
import dagger.hilt.android.HiltAndroidApp
import javax.inject.Inject

@HiltAndroidApp
class BaseApp:Application(){
    override fun onCreate() {
        super.onCreate()
    }
}